x/programs/rust/wasmlanche-sdk/src/program.rs

```diff
@@ -68,11 +68,16 @@ impl<K> Program<K> {
             max_units,
         };
 
-        let args_bytes = borsh::to_vec(&args).map_err(|_| StateError::Serialization)?;
+        let args_bytes = borsh::to_vec(&args).expect("failed to serialize args");
```

Comment:
Args will always serialize so this is fine.

Reply:
They could be user-defined, I would tend to think that it would be better to return an `ExecutionFailure` here, a serialization error could be added in the future if needed